src/Uno.UWP/Storage/Streams/DataWriter.cs

```diff
@@ -276,5 +283,33 @@ private void WriteChunkToUnstoredBuffer(byte[] chunk)
 			}
 			_memoryStream.Write(chunk, 0, chunk.Length);
 		}
+
+
+		public DataWriterStoreOperation StoreAsync()
+		{
+			var dwStore = new DataWriterStoreOperation();
+			dwStore.Task = new Task<uint>(() =>
```

Comment:
I didn't notice originally - in this case you might want to use `Task.Run` instead, as that can take argument of `Func<Task>` so instead of having to `Wait()` inside, you could then properly use `await` for the `WriteAsync` and `FlushAsync` calls 👍